src/components/button/__snapshots__/index.test.tsx.snap

```diff
@@ -2,6 +2,6 @@
 
 exports[`Button component should render successfully 1`] = `
 <button
-  className="c-btn "
+  className="c-btn undefined"
```

Comment:
You should check for existing `className` prop before building the attribute (yes, it's my fault).